scripts/init_serverless_sdk.py

```diff
@@ -11,15 +11,25 @@
 
 import sentry_sdk
 from sentry_sdk._types import MYPY
+from sentry_sdk.utils import Dsn
 from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
 
 if MYPY:
     from typing import Any
 
 
+# TODO(neel) maybe conditional on extension presence/layer version?
```

Comment:
We should add a check if the extension is present and only in this case change the dsn.

Ways to do this:
- checking for the /opt/extensions folder (hackish)
- introduce env var that defaults to "extension present". this way we do not need to set it in existing lambda functions, we need to set it only for cases where there is the layer but no extension.  (should this case ever happen in the future?)